### PR TITLE
[15.0][MIG] website_sale_order_type: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
TT37013

@chienandalu @CarlosRoca13 please review!

### CODE REVIEW

File: website_sale_order_type/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,5 +1,7 @@
 * Simone Rubino <simone.rubino@agilebg.com>
+* João Marques
 
 * `Tecnativa <https://www.tecnativa.com>`_:
 
-    * João Marques
```
Comment (CarlosRoca13) [inline website_sale_order_type/readme/CONTRIBUTORS.rst]: João was working on tecnativa when he CONTRIBUTE on this module so don't remove it from Tecnativa :smile:

**Review by CarlosRoca13 — APPROVED** [review]
Just a minor comment

File: website_sale_order_type/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (stefan-tecnativa) [reply website_sale_order_type/readme/CONTRIBUTORS.rst]: Thanks for the comment.

**Review by chienandalu — APPROVED** [review]
Thanks @stefan-tecnativa ! :+1: 

@pilarvargas-tecnativa give your final approval, please :wink:

**Review by pilarvargas-tecnativa — APPROVED** [review]
Approved!

**Pull request merged**